ydb/core/kqp/provider/yql_kikimr_datasink.cpp

```diff
@@ -186,9 +186,9 @@ class TKiSinkIntentDeterminationTransformer: public TKiSinkVisitorTransformer {
     }
 
     TStatus HandleDropObject(TKiDropObject node, TExprContext& ctx) override {
-        ctx.AddError(TIssue(ctx.GetPosition(node.Pos()), TStringBuilder()
-            << "DropObject is not yet implemented for intent determination transformer"));
-        return TStatus::Error;
+        Y_UNUSED(node);
+        Y_UNUSED(ctx);
+        return TStatus::Ok;
```

Comment:
The same (i.e. nothing) is done in [HandleCreateObject](https://github.com/ydb-platform/ydb/blob/414cd61a450b0e82a9196e7c0dd20fcc555055d5/ydb/core/kqp/provider/yql_kikimr_datasink.cpp#L179) 12 lines above.